src/app/dashboard/components/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,24 @@
+<div>
```

Comment:
We should decide whether we are going to be using inline templates or separate template files and then go with one approach